nri/index.html

```diff
@@ -1 +1,236 @@
 <!DOCTYPE html>
+ <html lang="ja">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>NRIの強み：ビジネスモデルの特徴</title>
+  <script src="https://kit.fontawesome.com/58440588d4.js" crossorigin="anonymous"></script>
+</head>
+<body>
+  <header>
+    <div class="container">
+      <div class="header-left">
+        <img src="http://or2.mobi/data/img/280135.png" alt="NRIのロゴ">
+      </div>
+      <div class="header-right">
+        <ul>
+          <li><a href="#">ナレッジ・インサイト</a></li>
+          <li><a href="#">ソリューション・サービス</a></li>
+          <li><a href="#">サステナビリティ</a></li>
+          <li><a href="#">ニュース</a></li>
+          <li><a href="#">企業情報</a></li>
+          <li><a href="#">株主・投資家情報(IR)</a></li>
+          <li><a href="#">採用情報</a></li>
+        </ul>
+      </div>
+      <div class="header-under-logo">
+        <ul>
+          <li>NRI トップ &gt;</li>
+          <li>企業情報 &gt;</li>
+          <li>NRIの強み：NRIのデジタルビジネス戦略</li>
+        </ul>
+      </div>
+      <div class="header-main">
+        <h2>NRIの強み</h2>
+        <h1>NRIのデジタルビジネス戦略</h1>
+      </div>
+      <div class="header-sns">
+        <p>share</p>
+        <ul>
+          <li><i class="fab fa-facebook-f"></i>
+          <li><i class="fab fa-twitter"></i>
+          <li><i class="fab fa-linkedin-in"></i>
+        </ul>
+      </div>
+    </div>
+  </header>
+  <div class="top-wrapper">
+    <div class="strategy-of-business">
+      <h2>NRIのデジタルビジネス戦略</h2>
+      <p>NRI では、企業の DX に向けた取り組みについて、既存ビジネスの業務プロセス変革に寄与する DX を「DX1.0」、デジタルで新しいビジネスモデルそのものを生み出す DX を「DX2.0」と定義しています。</p>
+      <br>
+      <p>DX を実現するためには、業務プロセス変革やビジネスモデル創造、そして、それらを実現する IT を同時並行で、共に議論しながら、かつ、スピーディーに実装できる戦略パートナーが必要です。NRI は、コンサルティングと IT ソリューションの人材と機能を併せ持ち、お客さまと併走し、お客さまと仮説検証を繰り返しながらビジネスを創出する「コンソリューション」の提供を強みとして、DX の実現を推進します。</p>
+      <img src="http://or2.mobi/data/img/280137.png" alt="NRIのデジタルビジネス戦略の図">
+      <a href="#">デジタルが拓く近未来（NRIのDX）<br>(4.06 MB)<i class="far fa-file-pdf"></i></a>
+      <p> &lt;<a href="#">ビジネスモデルの特徴</a>　<a href="#">人材が支えるNRI</a>&gt;</p>
```

Comment:
```suggestion
      <p><a href="#">ビジネスモデルの特徴</a>　<a href="#">人材が支えるNRI</a></p>
```
こういうの<>はHTMLに書かかないっぽいので除去しましょう！

Reply:
61008d1c596c9005f3c80122c28c23a37fdb8298
このコミットで対応しました。